vaadin-spring-tests/test-spring-security-fusion/src/main/java/com/vaadin/flow/spring/fusionsecurity/Application.java

```diff
@@ -2,13 +2,19 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import com.vaadin.flow.component.dependency.NpmPackage;
 import com.vaadin.flow.component.page.AppShellConfigurator;
 import com.vaadin.flow.server.PWA;
 import com.vaadin.flow.theme.Theme;
 
+
 @PWA(name = "Spring Security Helper Test Project", shortName = "SSH Test")
 @SpringBootApplication
 @Theme("spring-security-test-app")
+@NpmPackage(value = "@adobe/lit-mobx",
+            version = "2.0.0-rc.4")
+@NpmPackage(value = "mobx",
+            version = "6.1.5")
```

Comment:
For some reason 17 was using `"mobx": "^6.2.0"` if that has anything on the source compatibility.